docs/commands/InModuleScope.mdx

```diff
@@ -73,6 +73,56 @@ the PowerShell session, because the module only exported
 Using InModuleScope allowed this call to
 "PrivateFunction" to work successfully.
 
+### EXAMPLE 2
+
+```powershell
+# The script module:
+
+function PublicFunction
+
+    # Does something
+
+}
+
+function PrivateFunction ($MyParam)
+
+    return $MyParam
+}
+
+Export-ModuleMember -Function PublicFunction
+
+# The test script:
+
+Describe 'Testing MyModule' {
+    BeforeAll {
+        Import-Module MyModule
+    }
+
+    It 'passing in parameter' {
+        $SomeVar = 123
+      
+            $MyVar | Should -Be 123
+        }
+    }
```

Comment:
Corrupted. Source is:
```
        It 'passing in parameter' {
            $SomeVar = 123
            InModuleScope 'MyModule' -Parameters @{ MyVar = $SomeVar } {
                $MyVar | Should -Be 123
            }
        }
```

This doesn't have a trailing empty line, bug in the new code?

Reply:
Yes, very likely a bug. I just noticed something similar. Thanks for double checking!

Reply:
Resolved as well I think